public/usage-examples/graphics/draw_line_on_window/draw_line_on_window-1-simple.py

```diff
@@ -0,0 +1,21 @@
+from splashkit import *
+
+#Create Window
+start = open_window("draw line on window", 600, 600)
+clear_screen(color_black())
+
+
+#Draw line on window - param (Window, Color, x1, y1, x2, y2)
```

Comment:
Check the python comments have a space after `#`.

And as above - improve this comment.